src/arrayMethodSort.js

```diff
@@ -4,9 +4,20 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (callback = (a, b) => String(a) > String(b)) {
     // write code here
-  };
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (callback(this[i], this[j]) > 0) {
+          const firstValue = this[i];
+
+          this[i] = this[j];
+          this[j] = firstValue;
+        }
```

Comment:
The swapping of elements can be simplified by using array destructuring: `[this[i], this[j]] = [this[j], this[i]]`.